### ISSUE TITLE
Visualization problems with IGV

### ISSUE THREAD

**Issue by jennaj** [galaxyproject/usegalaxy-playbook#122]
Visualizing VCF files in IGV is problematic (again, or still). BAMs have a painfully slow connection/data transfer. 

**Workaround for end-users:** Download the VFC file locally, then upload it directly to the target display application. 

See related ticket about VCF visualization problems at iobio https://github.com/galaxyproject/galaxy/issues/4890

**First problem:**

IGV indexed genome (hg19) and a VCF (also hg19). Opened IGV locally and used the link inside the dataset. The data preparation step for the `tabix_file` never seems to finish.

BUT - the "track" shows up in IGV with a spinning wheel that never completes.

Start...

<img width="822" alt="screen shot 2018-05-11 at 3 24 32 pm" src="https://user-images.githubusercontent.com/8999735/39949415-8889b484-552f-11e8-9076-3e9c754761cb.png">

After a wait and clicking on "refresh". The timestamps are the same??

<img width="542" alt="screen shot 2018-05-11 at 3 58 20 pm" src="https://user-images.githubusercontent.com/8999735/39950167-481de00a-5534-11e8-8010-e42c90118ed4.png">

**Second problem:**

I then attempted to view a BAM. The BAM dataset loaded immediately and is viewable at first, but any clicks to zoom, etc, trigger more waits. 

**Third problem:**

When I backed out and tried to visualize again, Galaxy seemed to have some sort of cached file and thinks the genome is already prepared (??). If the data was not fully indexed, it shouldn't stick around to be used again. There are no warnings or obvious problems that something is going wrong, just no visualization in IGV

**Fourth problem:**

Completely remove IVG, reinstalled the latest version (2_4_10), then used the IGV methods to create a genome file and upload it there (this genome was not already indexed in IGV, not even in the tools). Creating a genome file is fast, uploading it is very very slow (lots of spinning). This was a very small genome. 

VCF - view local IGV - never seems to load or be available for visualization. This is after more spinning/waiting.

BAM - view local IGV - same result as when using a native IGV index. Displays at first, but changing position or most other functions, the IGV application freezes, spins, freezes, displays, click anything and this repeats.

**Is Galaxy or IGV the problem?**

Is it with local IGV application itself or the connection between it and Galaxy, or some configuration between the two that is problematic right now. Others have reported problems visualizing VFC data at IGV.

**Comment by jennaj** [comment]
When I tested the workaround, the data for hg19 all suddenly loaded into IGV and became accessible. I was able to change genome positions without a wait. I then switched genomes (to the IGV created/uploaded custom genome), states that tracks are present at the bottom of the app, but none are actually in the side panel. Switched back to hg19, same thing, tracks gone, are listed as being present at the bottom of the app, but none are displayed, including the downloaded/uploaded VCF. And no tracks are available under the "Tracks" masthead menu.

IGV used to work fine. It seems really buggy now.
